Examples/Python/src/Geometry.cpp

```diff
@@ -274,6 +275,37 @@ void addExperimentalGeometry(Context& ctx) {
         .def(py::init<std::vector<std::shared_ptr<Surface>>>());
   }
 
+  {
+    using Range1D = Acts::RangeXD<1u, Acts::ActsScalar>;
+    using KdtSurfaces1D = Acts::Experimental::KdtSurfaces<1u, 100u>;
+    using KdtSurfacesProvider1D =
+        Acts::Experimental::KdtSurfacesProvider<1u, 100u>;
+
+    py::class_<Range1D>(m, "Range1D")
+        .def(py::init([](const std::array<Acts::ActsScalar, 2u>& irange) {
+          Range1D range;
+          range[0].shrink(irange[0], irange[1]);
+          return range;
+        }));
```

Comment:
Why is Range1D not the C++ type Range1D but the RangeXD?

And how does this compile? `shrink` is defined on the *actual* Range1D, not on RangeXD.